app/views/grids/_notes.html.erb

```diff
@@ -95,6 +103,22 @@
   });
   //setupGridSorters(".<%= className %>-<%= randomSeed %>"); Should order the items by time. Commented out because it reorders the items each time show more/less is clicked
 })()
+
+function gridLinkModal(selector, href) {
+  var current_user = $('ul.nav.navbar-nav.navbar-right').children('li:nth-of-type(2)').text().trim() !== "Login"
+  if(!current_user) {
+    
+    var anchor = $(selector).find('a[href="' + href + '"]');
+    anchor.attr({
+      'href': '',
+      'data-toggle': 'modal',
+      'data-target': '#loginModal'
+    });
+    anchor.on('click', function() {
```

Comment:
Here, why are we requesting JSON, and why are we doing so upon clicking the button, as opposed to clicking something in the modal form? But I am guessing that I'm misunderstanding what this call does and where it goes... 

Thanks for helping me understand!